building-epics/cross-compile-epics-and-a-ioc-to-an-old-x86-linux.md

```diff
@@ -0,0 +1,378 @@
+# How To cross compile EPICS and a IOC to an old x86 Linux system
+
+## Introduction
+
+I was given the task of developing a IOC which should run in a x86 PC with an old Linux distribution.
+My development machine was a x86_64 PC running Ubuntu 12.04.
+EPICS does a great job cross compiling from a 64-bits host to a 32-bits target if both have compatible versions of glibc,
+binutils,
+kernel,
+etc.
+In my case,
+however,
+my target had much older versions. I considered two different solutions:
+
+1.  Create a Virtual Machine and install the target's Linux distribution. From the Virtual Machine,
+compile EPICS and my IOC,
+and then run the IOC in the target
+2.  Build a toolchain configured for my target and use that toolchain to compile both EPICS and the IOC.
```

Comment:
You can indent the numbered list (although it's not specified in the style guide, so either is fine).

```suggestion
1.  Create a Virtual Machine and install the target's Linux distribution.
    From the Virtual Machine,
    compile EPICS and my IOC,
    and then run the IOC in the target
2.  Build a toolchain configured for my target
    and use that toolchain to compile both EPICS and the IOC.
```